<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>681</width>
      <height>444</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>当前状态</string>
     </attribute>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>590</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>414</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string>室内温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getTempBtn">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>获取</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="temperatureTB">
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>192</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>getTempBtn</zorder>
      <zorder>label_3</zorder>
      <zorder>temperatureTB</zorder>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>279</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>客厅灯：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LEDonBtn">
         <property name="text">
          <string>开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LEDoffBtn">
         <property name="text">
          <string>关</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>系统参数</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>650</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>520</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="systemInfoBtn">
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>系统信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>66</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>CPU频率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="CPUfreqTB">
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>66</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>空闲内存：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="memspaceTB">
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="maximumSize">
          <size>
           <width>66</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>剩余磁盘：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="diskspaceTB">
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>349</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>系统升级：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kernelUpdateBtn">
         <property name="text">
          <string>Kernel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rootfsUpdateBtn">
         <property name="text">
          <string>Rootfs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="userdataUpdateBtn">
         <property name="text">
          <string>Userdata</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>571</width>
        <height>20</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>硬件版本：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hwVersionLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>软件版本：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="swVersionLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>line</zorder>
     <zorder>line_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action123">
   <property name="text">
    <string>123</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
